worlds/ahit/__init__.py

```diff
@@ -0,0 +1,324 @@
+from BaseClasses import Item, ItemClassification, Tutorial, Location, MultiWorld
+from .Items import item_table, create_item, relic_groups, act_contracts, create_itempool, get_shop_trap_name, \
+    calculate_yarn_costs
+from .Regions import create_regions, randomize_act_entrances, chapter_act_info, create_events, get_shuffled_region
+from .Locations import location_table, contract_locations, is_location_valid, get_location_names, TASKSANITY_START_ID, \
+    get_total_locations
+from .Rules import set_rules
+from .Options import AHITOptions, slot_data_options, adjust_options
+from .Types import HatType, ChapterIndex, HatInTimeItem
+from .DeathWishLocations import create_dw_regions, dw_classes, death_wishes
+from .DeathWishRules import set_dw_rules, create_enemy_events
+from worlds.AutoWorld import World, WebWorld
+from typing import List, Dict, TextIO
+from worlds.LauncherComponents import Component, components, icon_paths, launch_subprocess, Type
+from Utils import local_path
+
+
+def launch_client():
+    from .Client import launch
+    launch_subprocess(launch, name="AHITClient")
+
+
+components.append(Component("A Hat in Time Client", "AHITClient", func=launch_client,
+                            component_type=Type.CLIENT, icon='yatta'))
+
+icon_paths['yatta'] = local_path('data', 'yatta.png')
+
+
+class AWebInTime(WebWorld):
+    theme = "partyTime"
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide for setting up A Hat in Time to be played in Archipelago.",
+        "English",
+        "ahit_en.md",
+        "setup/en",
+        ["CookieCat"]
+    )]
+
+
+class HatInTimeWorld(World):
+    """
+    A Hat in Time is a cute-as-peck 3D platformer featuring a little girl who stitches hats for wicked powers!
+    Freely explore giant worlds and recover Time Pieces to travel to new heights!
+    """
+
+    game = "A Hat in Time"
+    data_version = 1
```

Comment:
this is deprecated and should be removed

Reply:
done